index.bs

```diff
@@ -3464,7 +3464,7 @@ The W3C Recommendation Track</h3>
 	W3C <em class="rfc2119">may</em> <a href="#tr-end">end work on a technical report</a> at any time.
 
 	As described in [[#transition-reqs]],
-	the [=Team=] will decline a request to advance in maturity stage
+	the [=Team=] will decline a request to advance in the maturity stage
```

Comment:
Should be either `advance in` or `advance the`; should not be both `advance in the`. I prefer the former.
```suggestion
	the [=Team=] will decline a request to advance in maturity stage
```